src/instance/InstanceController.ts

```diff
@@ -1272,3 +1281,50 @@ export class InstanceController {
     }
   }
 }
+
+@ApiTags('Instances')
+@Controller('instances')
+export class InstancesController {
+  constructor(private readonly instanceService: InstanceService) {}
+
+  @Get()
+  @ApiOperation({summary: 'Get all instances.'})
+  @ApiResponse({
+    schema: {
+      example: {
+        backend: 'string',
+        clientId: 'string',
+        instanceId: '3fa85f64-5717-4562-b3fc-2c963f66afa6',
+        name: 'string',
+      },
+    },
+    status: 200,
+  })
+  @ApiResponse(status500description)
+  async getInstances(@Res() res: Response): Promise<void> {
+    const instances = this.instanceService.getInstances();
+
+    if (!Object.keys(instances).length) {
+      res.json({});
```

Comment:
```suggestion
      return res.json({});
```

Reply:
We are not returning the response anywhere else